packages/babel-preset-env/test/fixtures/plugins-integration/class-features-node-12/options.json

```diff
@@ -0,0 +1,7 @@
+{
+  "validateLogs": true,
+  "targets": "node 12",
```

Comment:
```suggestion
  "targets": "node 12.0",
```

Reply:
Done because it's better to be explicit, but in this particular case it doesn't matter because the support is the same for all the Node.js 12 versions (it's in Node.js 14 that it matters)